tier2/checklist.md

```diff
@@ -0,0 +1,413 @@
+# DSAC OSPO Outbound Review Checklist
+## Tier 2: Close Collaboration
+### Instructions
+This is a review process to approve CMS-developed software to be released open source at [github.cms.gov](github.cms.org).
```

Comment:
I think this should link to github.cms.gov instead of github.cms.org since that shows up as a broken link for me.